x-pack/plugins/upgrade_assistant/__jest__/client_integration/overview/backup_step/backup_step.test.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setupEnvironment } from '../../helpers';
+import { OverviewTestBed, setupOverviewPage } from '../overview.helpers';
+
+describe('Overview - Backup Step', () => {
+  let testBed: OverviewTestBed;
+  const { server } = setupEnvironment();
+
+  beforeEach(async () => {
+    testBed = await setupOverviewPage();
+    testBed.component.update();
+  });
+
+  afterAll(() => {
+    server.restore();
+  });
+
+  describe('On-prem', () => {
+    test('Shows link to Snapshot and Restore', () => {
```

Comment:
Was wondering if we should also assert on the url of the link in order to be more explicit about where we are expecting users to go 🤔

Reply:
Thanks for suggesting this! I added a test, but tried to structure the mocks to avoid testing the locator itself. Love to hear your thoughts on this.